From bf49b168333f5b19cbb6e8ca2aaf79060e166e41 Mon Sep 17 00:00:00 2001
From: Bo Shen <voice.shen@atmel.com>
Date: Thu, 30 Oct 2014 14:23:42 +0800
Subject: [PATCH] sama5d4ek: remove nand bus configuration

As the commit: 830a08b remove dynamically configure nand flash
bus width, so no need nand bus configuration function anymore.

Signed-off-by: Bo Shen <voice.shen@atmel.com>
---
 board/sama5d4ek/sama5d4ek.c | 15 ---------------
 board/sama5d4ek/sama5d4ek.h |  1 -
 2 files changed, 16 deletions(-)

diff --git a/board/sama5d4ek/sama5d4ek.c b/board/sama5d4ek/sama5d4ek.c
index 754f43c..278c0f9 100644
--- a/board/sama5d4ek/sama5d4ek.c
+++ b/board/sama5d4ek/sama5d4ek.c
@@ -882,19 +882,4 @@ void nandflash_hw_init(void)
 		| AT91C_SMC_MODE_TDF_CYCLES(1),
 		(ATMEL_BASE_SMC + SMC_MODE3));
 }
-
-void nandflash_config_buswidth(unsigned char buswidth)
-{
-	unsigned int mode;
-
-	mode = readl(ATMEL_BASE_SMC + SMC_MODE3);
-
-	mode &= ~AT91C_SMC_MODE_DBW;
-	if (buswidth == 0)	/* 8 bits bus */
-		mode |= AT91C_SMC_MODE_DBW_8;
-	else
-		mode |= AT91C_SMC_MODE_DBW_16;
-
-	writel(mode, (ATMEL_BASE_SMC + SMC_MODE3));
-}
 #endif /* #ifdef CONFIG_NANDFLASH */
diff --git a/board/sama5d4ek/sama5d4ek.h b/board/sama5d4ek/sama5d4ek.h
index 682a73f..df830d7 100644
--- a/board/sama5d4ek/sama5d4ek.h
+++ b/board/sama5d4ek/sama5d4ek.h
@@ -196,7 +196,6 @@
 extern void hw_init(void);
 
 extern void nandflash_hw_init(void);
-extern void nandflash_config_buswidth(unsigned char busw);
 
 extern void at91_spi0_hw_init(void);
 
-- 
1.9.1

